streamlit_app/app.py

```diff
@@ -12,7 +12,10 @@
 
 # src should sit in the same level as /streamlit_app
 curr_dir = os.getcwd()
+
 parent_dir = os.path.dirname(curr_dir)
+#parent_dir = os.path.dirname(curr_dir) + '/memary' #Use this if error: src not found. Also move the '/streamlit_app/data' folder to the 'memary' folder, outside the 'src' folder.
```

Comment:
Thanks for this detailed explanation!

Reply:
We will include this in the instructions. Thanks!

Reply:
This is great. Thanks!